### PR TITLE
Javascript2-Week1 homework

### PR DESCRIPTION
**Pull request by gsismandnm**

### CODE REVIEW

File: javascript/javascript2/week1/shortestWord.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+// Find the shortest word
+
+const danishWords = ['bil', 'plante', 'kaffe', 'bog', 'ø', 'planetarium'];
+//notThisFunctionName(danishWords); // returns 'ø'
+
+  function findShortestWord(danishWords) {    
+    var shortest = danishWords.reduce((shortestWord, currentWord) => {
```
Comment (jcg-codes) [inline javascript/javascript2/week1/shortestWord.js]: Remember to use `const` not `var`

File: javascript/javascript2/week1/shortestWord.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+      return currentWord.length < shortestWord.length ? currentWord : shortestWord;
+    }, danishWords[0]);
```
Comment (jcg-codes) [inline javascript/javascript2/week1/shortestWord.js]: You don't need this array here `danishWords[0]`

File: javascript/javascript2/week1/shortestWord.js [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+    return shortest;
+  }
+
+  console.log(danishWords);
+  console.log(findShortestWord(danishWords));
```
Comment (jcg-codes) [inline javascript/javascript2/week1/shortestWord.js]: Good job, but try to give clear message in console log ex: `console.log('short word is: ', findShortestWord(danishWords));`

File: javascript/javascript2/week1/animalName.js [hunk 4]
```diff
@@ -0,0 +1,35 @@
+// Spirit animal name generator
+
+const p = document.querySelector("p"); // for show the name
+const input = document.getElementById("inputName"); // for write the name input text
+const btn = document.querySelector("button"); // make a button
+
+// make a function  that will return the input name
+
+
+btn.addEventListener("click", function clickOne(){
```
Comment (jcg-codes) [inline javascript/javascript2/week1/animalName.js:10]: Great job, now you are half way. Remember, a user need to have 3 options for checking animal Spirit (on Click, on hover and on input) so far you made one option. Here is a tip: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/radio

File: javascript/javascript2/week1/shortestWord.js (hunk 1 above)
Comment (gsismandnm) [reply javascript/javascript2/week1/shortestWord.js]: Yes. I will do.

File: javascript/javascript2/week1/shortestWord.js (hunk 2 above)
Comment (gsismandnm) [reply javascript/javascript2/week1/shortestWord.js]: I will delete it.

File: javascript/javascript2/week1/shortestWord.js (hunk 3 above)
Comment (gsismandnm) [reply javascript/javascript2/week1/shortestWord.js]: Ok.

**Pull request merged**